packages/@sanity/desk-tool/src/panes/document/documentPanel/DocumentPanel.tsx

```diff
@@ -63,6 +64,7 @@ export const DocumentPanel = function DocumentPanel(props: DocumentPanelProps) {
   const headerRect = useElementRect(headerElement)
   const portalRef = useRef<HTMLDivElement | null>(null)
   const [documentScrollElement, setDocumentScrollElement] = useState<HTMLDivElement | null>(null)
+  const [isReferencedDocument] = useDocumentsIsReferenced(documentId)
```

Comment:
I believe this will end up creating two listeners for the same information - I would prefer that we either receive the `isReferencedDocument` through a prop, or as stated above, through the `DocumentPaneContext`